vendor/.editorconfig

```diff
@@ -0,0 +1,2 @@
+# do not use any .editorconfig files in upper level dir
+root = true
```

Comment:
This "root" I understand, we don't want to modify vendored files. But the 4 other roots I don't get at all. They seem to remove the editorconfig rules from our python and sql files.